rails/app/controllers/portal/offerings_controller.rb

```diff
@@ -126,27 +126,6 @@ def deactivate
     redirect_to :back
   end
 
-  def answers
-    @offering = Portal::Offering.find(params[:id])
-    authorize @offering
-    if @offering
-      learner = setup_portal_student
```

Comment:
Did you check if this was called elsewhere? I probably has to be otherwise I think the learner will never be created for the student.

Reply:
Hm, that's the `answers` path of the offering controller, so I'm assuming it's no longer necessary. 

But if you mean general creation of the learner object - my understanding is that at this point we mostly use this model to track whether students started an offering and when (last run in the UI).

I manually checked that - I logged in as a new student and ran one of the offerings. I could still see the "Last Run" date both on the student status page and on the teacher status page under specific assignment. So, I'm assuming that the learner object has been created as it should.